docs-src/content/functions/coll.yml

```
@@ -136,6 +136,37 @@ funcs:
       - |
         $ gomplate -i '{{ .books | jsonpath `$..works[?( @.edition_count > 400 )].title` }}' -c books=https://openlibrary.org/subjects/fantasy.json
         [Alice's Adventures in Wonderland Gulliver's Travels]
+  - name: coll.JQ
+    alias: jsonpath
+    description: |
+      Filters an input object or list using the JQ language implemented by gojq (https://github.com/itchyny/gojq).
+
+      Any object or list may be used as input. The output depends somewhat on the expression; if multiple items are matched, an array is returned.
+
+      JQ filter expressions can be tested at https://jqplay.org/
+
+      [jq Manual]: https://stedolan.github.io/jq/manual/
+      [gojq differences to jq]: https://github.com/itchyny/gojq#difference-to-jq
```

these markdown footers aren't referenced anywhere, so are not useful. Maybe something like:

```suggestion
      See also:
      - [jq manual](https://stedolan.github.io/jq/manual/)
      - [gojq differences to jq](https://github.com/itchyny/gojq#difference-to-jq)
```

(See also the preview at https://deploy-preview-1585--gomplate-docs.netlify.app/functions/coll/#coll-jq)